### PR TITLE
Update PASQAL quickstart documentation

### PR DESCRIPTION
**Pull request by MohamedAminMallek**
Describe the required steps to submit a job to PASQAL using pulser sdk

### CODE REVIEW

**Comment by Jak-MS** [comment]
@MohamedAminMallek - can you agree to the CLA by following the instructions here: https://github.com/MicrosoftDocs/quantum-docs/pull/659#issuecomment-2414136585

We can't move forward without your sign off on that. thanks!

**Comment by MohamedAminMallek** [comment]
@microsoft-github-policy-service agree company="Pasqal"

**Comment by Jak-MS** [comment]
thanks @MohamedAminMallek 

@Bradben - 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: articles/quickstart-microsoft-provider-format.md [hunk 1]
```diff
@@ -284,83 +284,74 @@ try:
     import pulser
 except ImportError:
     !pip -q install pulser
-    !pip -q install pulser-core
 ```
 
 #### Create a quantum register
 
-1. First, load the required imports:
+1. Fresnel is the first PASQAL commercial neutral atoms quantum computer that you can target for your quantum jobs. To configure your quantum registers, you arrange an array of qubits. FRESNEL provides predefined trap layouts that you must utilize to create your quantum register.
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:291]: Is it Fresnel, FRESNEL, or Fresnel1? According to the docs, we have [Fresnel1](https://learn.microsoft.com/en-us/azure/quantum/provider-pasqal#fresnel1)

Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:291]:
```suggestion
1. Fresnel is the first PASQAL commercial neutral atoms quantum computer that you can target for your quantum jobs. To configure your quantum registers, you arrange an array of qubits. FRESNEL provides predefined trap layouts that you can utilize to create your quantum register.
```

File: articles/quickstart-microsoft-provider-format.md [hunk 2]
```diff
@@ -284,83 +284,74 @@ try:
...
+    devices = PasqalCloud().fetch_available_devices()
+    QPU = devices["FRESNEL"]
+
+    # List all available calibrated register layouts
+    for calibrated_register_layout in QPU.calibrated_register_layouts.keys():
+        print(calibrated_register_layout)
     ```
 
-1. PASQAL's QPU is made of neutral atoms trapped at well-defined positions in a lattice. To define your quantum registers you create an array of qubits on a lattice. For example, the following code creates a 4x4 square lattice of qubits:
+2. In this example, we will be using `TriangularLatticeLayout(61, 5.0µm)` as the trap layout. You can visualize the chosen layout.
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:304]:
```suggestion
1. Next, you define the layout for your qubit register. In this example, you use a `TriangularLatticeLayout(61, 5.0µm)` as the trap layout. 
```

File: articles/quickstart-microsoft-provider-format.md [hunk 3]
```diff
@@ -284,83 +284,74 @@ try:
...
-    square *= 5
-    
-    qubits = dict(enumerate(square))
-    reg = Register(qubits)
+    layout = QPU.calibrated_register_layouts[
+        "TriangularLatticeLayout(61, 5.0µm)"
+    ]
+    layout.draw()
+    ````
+    :::image type="content" source="media/provider-format-pasqal-layout.png" alt-text="Trap layout":::
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:312]:
```suggestion
    The following image shows the display of the chosen layout. 
    
    :::image type="content" source="media/provider-format-pasqal-layout.png" alt-text="Diagram of the layout chosen for the qubit register. The layout shows 60 points in an array of 40 times 30 micrometers.":::
```

File: articles/quickstart-microsoft-provider-format.md [hunk 4]
```diff
@@ -284,83 +284,74 @@ try:
...
+
+3. In this example we will select 7 traps using their ids to define a quantum register of 7 qubits.
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:314]:
```suggestion
1. Then, you define the qubit register selecting a set of traps from the layout. In this example, the layout has 60 traps and you select 7 traps using their ids to define a quantum register of 7 qubits.
```

File: articles/quickstart-microsoft-provider-format.md [hunk 5]
```diff
@@ -284,83 +284,74 @@ try:
...
+
+    ```python
+    reg = layout.define_register(*[30, 21, 26, 35, 39, 34, 25])
     reg.draw()
     ```
-
-    :::image type="content" source="media/provider-format-pasqal-array.png" alt-text="Plot of a 4x4 square lattice with 16 qubits.":::
+    :::image type="content" source="media/provider-format-pasqal-register.png" alt-text="Quantum register":::
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:320]:
```suggestion
    The following image shows the final display of the qubit register.
    
    :::image type="content" source="media/provider-format-pasqal-register.png" alt-text="Diagram showing the final quantum register after selecting 7 points from the 60 point-layout.":::
```

File: articles/quickstart-microsoft-provider-format.md [hunk 6]
```diff
@@ -284,83 +284,74 @@ try:
...
 
 #### Write a pulse sequence
 
 The neutral atoms are controlled with laser pulses. The Pulser SDK allows you to create pulse sequences to apply to the quantum register.
 
-1. First, you need to set up a pulse sequence, and declare the channels that will be used to control the atoms. For example, the following code declares two channels: `ch0` and `ch1`.
+1. Define the pulse sequence attributes by declaring the channels that will be used to control the atoms. For example, the following code declares one channel: `ch0`.
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:326]:
```suggestion
1. First,  you define the pulse sequence attributes by declaring the channels that will be used to control the atoms. To create a `Sequence`, you need to provide a `Register` instance along with the device where the sequence will be executed. For example, the following code declares one channel: `ch0`.
```

File: articles/quickstart-microsoft-provider-format.md [hunk 7]
```diff
@@ -284,83 +284,74 @@ try:
...
+
+- To create a Sequence, you need to provide a Register instance along with the device where the sequence will be executed. You can use the Fresnel QPU device we created earlier or import a virtual device from Pulser for more flexibility. The use of a VirtualDevice allows for sequence creation that is less constrained by device specifications, making it suitable for execution on an emulator. For more information, refer to the [Pulser documentation](https://pulser.readthedocs.io/en/stable/tutorials/creating.html#2.-Initializing-the-Sequence).
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:328]:
```suggestion
   > [!NOTE]
   > You can use the `QPU = devices["FRESNEL"]`  device or import a virtual device from Pulser for more flexibility. The use of a `VirtualDevice` allows for sequence creation that is less constrained by device specifications, making it suitable for execution on an emulator. For more information, see [Pulser documentation](https://pulser.readthedocs.io/en/stable/tutorials/creating.html#2.-Initializing-the-Sequence).
```

File: articles/quickstart-microsoft-provider-format.md [hunk 8]
```diff
@@ -284,83 +284,74 @@ try:
...
-The image shows the simple and the complex pulse.
+    amp_wf = BlackmanWaveform(1000, np.pi)
+    det_wf = RampWaveform(1000, -5, 5)
+    pulse = Pulse(amp_wf, det_wf, 0)
+    seq.add(pulse, "ch0")
 
-:::image type="content" source="media/provider-format-pasqal-pulser.png" alt-text="Plot of the simple and the complex pulse.":::
+    seq.draw()
+    ```
+    :::image type="content" source="media/provider-format-pasqal-sequence.png" alt-text="Pulse sequence":::
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:354]:
```suggestion
    The following image shows the pulse sequence. 
    :::image type="content" source="media/provider-format-pasqal-sequence.png" alt-text="Pulse sequence":::
```

File: articles/quickstart-microsoft-provider-format.md [hunk 9]
```diff
@@ -284,83 +284,80 @@ try:
     import pulser
 except ImportError:
     !pip -q install pulser
-    !pip -q install pulser-core
 ```
 
 #### Create a quantum register
 
-1. First, load the required imports:
+1. First, you create a 'devices' object to import the PASQAL quantum computer target, Fresnel. FRESNEL provides predefined trap layouts that you can utilize to create your quantum register. To configure your quantum registers, you arrange an array of qubits.
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:291]: Is it Fresnel, FRESNEL, or Fresnel1? Our docs show [Fresnel1](https://learn.microsoft.com/en-us/azure/quantum/provider-pasqal#fresnel1)

File: articles/quickstart-microsoft-provider-format.md [hunk 10]
```diff
@@ -284,83 +284,80 @@ try:
...
-The image shows the simple and the complex pulse.
+    amp_wf = BlackmanWaveform(1000, np.pi)
+    det_wf = RampWaveform(1000, -5, 5)
+    pulse = Pulse(amp_wf, det_wf, 0)
+    seq.add(pulse, "ch0")
 
-:::image type="content" source="media/provider-format-pasqal-pulser.png" alt-text="Plot of the simple and the complex pulse.":::
+    seq.draw()
+    ```
+    The following image shows the pulse sequence.
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:359]: Could you describe a bit the pulse sequence image?

File: articles/quickstart-microsoft-provider-format.md [hunk 11]
```diff
@@ -384,40 +380,73 @@ def prepare_input_data(seq):
...
-    
-        return result
+        return job
     ```
 
     > [!NOTE]
-    > The time required to run a circuit on the QPU depends on current queue times. You can view the average queue time for a target by selecting the **Providers** blade of your workspace.
+    > The time required to run a job on the QPU depends on current queue times. You can view the average queue time for a target by selecting the **Providers** blade of your workspace.
+
+2. Submit the program to PASQAL.
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:399]:
```suggestion
1. Submit the program to PASQAL. Before submitting your code to real quantum hardware, you can test your code using the emulator  `pasqal.sim.emu-tn` as a target.
```

File: articles/quickstart-microsoft-provider-format.md [hunk 12]
```diff
@@ -384,40 +380,73 @@ def prepare_input_data(seq):
...
+If this is your first time submitting a job, we encourage you to test your code using the emulator as a target.
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:400]:
```suggestion
```

File: articles/quickstart-microsoft-provider-format.md [hunk 13]
```diff
@@ -384,40 +380,72 @@ def prepare_input_data(seq):
...
+    > The time required to run a job on the QPU depends on current queue times. You can view the average queue time for a target by selecting the **Providers** blade of your workspace.
+
+1. Submit the program to PASQAL. Before submitting your code to real quantum hardware, you can test your code using the emulator  `pasqal.sim.emu-tn` as a target.
 
-1. Submit the program to PASQAL. For example, you can submit the program to [PASQAL Emu-TN target](xref:microsoft.quantum.providers.pasqal#emu-tn).
+- QPU target name = `pasqal.qpu.fresnel`
+- Emulator target name = `pasqal.sim.emu-tn`
 
     ```python
     target = workspace.get_targets(name="pasqal.sim.emu-tn")
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:405]:
```suggestion
    target = workspace.get_targets(name="pasqal.sim.emu-tn") # Change to "pasqal.qpu.fresnel" to use Fresnel QPU
```

File: articles/quickstart-microsoft-provider-format.md [hunk 14]
```diff
@@ -384,40 +380,72 @@ def prepare_input_data(seq):
...
     ```
 
     > [!NOTE]
-    > The time required to run a circuit on the QPU depends on current queue times. You can view the average queue time for a target by selecting the **Providers** blade of your workspace.
+    > The time required to run a job on the QPU depends on current queue times. You can view the average queue time for a target by selecting the **Providers** blade of your workspace.
+
+1. Submit the program to PASQAL. Before submitting your code to real quantum hardware, you can test your code using the emulator  `pasqal.sim.emu-tn` as a target.
 
-1. Submit the program to PASQAL. For example, you can submit the program to [PASQAL Emu-TN target](xref:microsoft.quantum.providers.pasqal#emu-tn).
+- QPU target name = `pasqal.qpu.fresnel`
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:401]:
```suggestion
```

File: articles/quickstart-microsoft-provider-format.md [hunk 15]
```diff
@@ -384,40 +380,71 @@ def prepare_input_data(seq):
...
     ```
 
     > [!NOTE]
-    > The time required to run a circuit on the QPU depends on current queue times. You can view the average queue time for a target by selecting the **Providers** blade of your workspace.
+    > The time required to run a job on the QPU depends on current queue times. You can view the average queue time for a target by selecting the **Providers** blade of your workspace.
+
+1. Submit the program to PASQAL. Before submitting your code to real quantum hardware, you can test your code using the emulator  `pasqal.sim.emu-tn` as a target.
 
-1. Submit the program to PASQAL. For example, you can submit the program to [PASQAL Emu-TN target](xref:microsoft.quantum.providers.pasqal#emu-tn).
+- Emulator target name = `pasqal.sim.emu-tn`
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:401]:
```suggestion
```

File: articles/quickstart-microsoft-provider-format.md [hunk 16]
```diff
@@ -384,40 +380,70 @@ def prepare_input_data(seq):
...
-    
-        return result
+        return job
     ```
 
     > [!NOTE]
-    > The time required to run a circuit on the QPU depends on current queue times. You can view the average queue time for a target by selecting the **Providers** blade of your workspace.
+    > The time required to run a job on the QPU depends on current queue times. You can view the average queue time for a target by selecting the **Providers** blade of your workspace.
+
+1. Submit the program to PASQAL. Before submitting your code to real quantum hardware, you can test your code using the emulator  `pasqal.sim.emu-tn` as a target.
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:399]:
```suggestion
1. Submit the program to PASQAL. Before submitting your code to real quantum hardware, you can test your code using the emulator `pasqal.sim.emu-tn` as a target.
```

File: articles/quickstart-microsoft-provider-format.md [hunk 17]
```diff
@@ -384,40 +380,70 @@ def prepare_input_data(seq):
...
+    ```output
+    {
+        "1000000": 3, 
+        "0010000": 1, 
+        "0010101": 1
+    }
     ```
 
+#### Explore Advanced Emulator Features
+Our emulators offer advanced features that are not yet supported by Pasqal's QPU:
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:421]:
```suggestion
[PASQAL's emulator](xref:microsoft.quantum.providers.pasqal#emulator) offers advanced features that are not yet supported by Fresnel QPU:
```

File: articles/quickstart-microsoft-provider-format.md [hunk 18]
```diff
@@ -384,40 +380,70 @@ def prepare_input_data(seq):
...
+
+##### Custom register
+Organize your register in a custom way without any limitations from the pre-calibrated layouts. Note that this feature will soon be available on FRESNEL.
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:424]:
```suggestion

You can organize your register in a custom way without any limitations from the pre-calibrated layouts. For example, the following code creates a 4x4 square lattice of qubits: 
```

File: articles/quickstart-microsoft-provider-format.md [hunk 19]
```diff
@@ -384,39 +380,70 @@ def prepare_input_data(seq):
...
+    ```
+
+#### Explore Advanced Emulator Features
+Our emulators offer advanced features that are not yet supported by Pasqal's QPU:
+
+##### Custom register
+
+You can organize your register in a custom way without any limitations from the pre-calibrated layouts. For example, the following code creates a 4x4 square lattice of qubits: 
+
+- For example, the following code creates a 4x4 square lattice of qubits:
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:427]:
```suggestion
```

File: articles/quickstart-microsoft-provider-format.md [hunk 20]
```diff
@@ -384,39 +380,72 @@ def prepare_input_data(seq):
...
+    ```
+
+#### Explore Advanced Emulator Features
+Our emulators offer advanced features that are not yet supported by Pasqal's QPU:
+
+##### Custom register
+
+You can organize your register in a custom way without any limitations from the pre-calibrated layouts. For example, the following code creates a 4x4 square lattice of qubits: 
+
+
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:427]:
```suggestion
```

File: articles/quickstart-microsoft-provider-format.md [hunk 21]
```diff
@@ -384,39 +380,71 @@ def prepare_input_data(seq):
...
+        "1000000": 3, 
+        "0010000": 1, 
+        "0010101": 1
+    }
+    ```
+
+#### Explore Advanced Emulator Features
+[PASQAL's emulator](xref:microsoft.quantum.providers.pasqal#emulator) offers advanced features that are not yet supported by Fresnel QPU:
+
+##### Custom register
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:423]:
```suggestion
```

File: articles/quickstart-microsoft-provider-format.md [hunk 22]
```diff
@@ -384,39 +380,71 @@ def prepare_input_data(seq):
...
+        "0010101": 1
+    }
+    ```
+
+#### Explore Advanced Emulator Features
+
+[PASQAL's emulator](xref:microsoft.quantum.providers.pasqal#emulator) offers advanced features that are not yet supported by Fresnel QPU:
+
+
+You can organize your register in a custom way without any limitations from the pre-calibrated layouts. For example, the following code creates a 4x4 square lattice of qubits:
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:425]:
```suggestion
```

File: articles/quickstart-microsoft-provider-format.md [hunk 23]
```diff
@@ -384,39 +380,70 @@ def prepare_input_data(seq):
...
+    {
+        "1000000": 3, 
+        "0010000": 1, 
+        "0010101": 1
+    }
+    ```
+
+#### Explore Advanced Emulator Features
+
+[PASQAL's emulator](xref:microsoft.quantum.providers.pasqal#emulator) offers advanced features that are not yet supported by Fresnel QPU:
```
Comment (SoniaLopezBravo) [inline articles/quickstart-microsoft-provider-format.md:422]:
```suggestion
[PASQAL's emulator](xref:microsoft.quantum.providers.pasqal#emulator) offers advanced features that are not yet supported by Fresnel QPU. You can organize your register in a custom way without any limitations from the pre-calibrated layouts. For example, the following code creates a 4x4 square lattice of qubits:
```

**Review by SoniaLopezBravo — COMMENTED** [review]
Thanks! I left a couple of comments

**Review by SoniaLopezBravo — APPROVED** [review]

**Pull request merged**